src/controller/java/src/chip/devicecontroller/WriteAttributesCallback.java

```diff
@@ -18,12 +18,33 @@
 package chip.devicecontroller;
 
 import chip.devicecontroller.model.ChipAttributePath;
+import javax.annotation.Nullable;
 
 /** An interface for receiving write response. */
 public interface WriteAttributesCallback {
-  void onError(ChipAttributePath attributePath, Exception e);
 
-  void OnResponse(ChipAttributePath attributePath);
+  /**
+   * OnError will be called when an error occurs after failing to write
+   *
+   * - CHIP_ERROR_TIMEOUT: A response was not received within the expected response timeout.
```

Comment:
This documentation is fairly confusing from Java user's perspective. It's unclear how one would determine the error code given the output is just an `Exception` type. Is the exception actually a `ChipDeviceControllerException` (in which case the `errorCode` property could be checked)? Or something else? `StatusIB::InitFromChipError` is not applicable to the Java API.

Reply:
updated, thanks